### ISSUE TITLE
[IDEA] Detecting SMB

### ISSUE THREAD

**Issue by Psype** [thom-s/netsec-ps-scripts#1]
Hey there,

First of all, nice work ! 
According to the last line of the README, I'd suggest some script to detect (and disable?) SMB implementations in networks peripherials, such as NAS & so on.
SMBv1 is a breach by itself :
https://techcommunity.microsoft.com/t5/storage-at-microsoft/stop-using-smb1/ba-p/425858

And is like... mandatory with my 2015 NAS. I have to keep it outside of the network.

Cheers !

**Comment by thom-s** [comment]
Good idea! I'll see if I have some time to try that out, though you should be able to do that with nmap or a vulnerability scanner like Nessus in the meantime.

Thank you!
